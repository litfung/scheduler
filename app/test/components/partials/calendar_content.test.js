import React from 'react';
import { shallow, mount } from 'enzyme';
import { Provider } from 'react-redux';

import events from '../../__data__/events';
import groups from '../../__data__/groups';

import CalendarContentComp from '../../../src/client/components/partials/calendar/calendar_content';
import configureStore from '../../../src/helpers/configure_store';

let jsx, wrapper, wrapper2, toggleEventDetails;

beforeEach(() => {
  toggleEventDetails = jest.fn();
  
  wrapper = shallow(
    <Provider store={configureStore({ get: jest.fn() })}>
      <CalendarContentComp
        miniCalendar={false}
        year={2019}
        month={0}
        staticContext={undefined}
        toggleEventDetails={toggleEventDetails}
      />
    </Provider>
  );
});

test('Snapshot for CalendarContentComp Component', () => {
  wrapper = 
  expect(wrapper).toMatchSnapshot();
});
