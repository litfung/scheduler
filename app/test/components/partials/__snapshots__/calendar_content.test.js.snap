// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for CalendarContentComp Component 1`] = `
<Connect(CalendarContentComp)
  miniCalendar={false}
  month={0}
  toggleEventDetails={[MockFunction]}
  year={2019}
/>
`;
